tcms/testruns/static/testruns/js/get.js

```diff
@@ -172,6 +172,32 @@ function renderTestExecutions(testExecutions) {
 function renderAdditionalInformation(testExecutions, testExecutionCaseIds) {
     $('.test-executions-count').html(testExecutions.length);
 
+    testExecutions.forEach(testExecution => {
+        // TODO: jsonRPC('TestExecution.get_comments', [testExecution.id], comments => {
+        const comments = []
+        const testExecutionRow = $(`.test-execution-${testExecution.id}`)
+        const commentsRow = testExecutionRow.find('.comments')
+        comments.forEach(comment => {
+            commentsRow.append(`<div class="comment">${comment.user_name}: ${comment.comment}</div>`)
```

Comment:
I know, I know, will remove this once I have better idea for the design. will leave it until then for testing purposes